tests/css_case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -89,15 +91,17 @@ $KeepScssVar: val;
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 // Convention in this test file:
 // - The case should be preserved for things prefixed with "Keep".
-// - The case should always be preserved for function names and property keywords.
+// - The case should always be preserved for element names and attribute names
+//   in selectors, as well as function names and property keywords.
 // - Other things should mostly be lowercase.
 // - The \`/*:*/\` comments are just to bust the \`isLikelySCSS\` check.
```

Comment:
Unrelated to this PR, but can we use `parser: "less"` instead of the comments now?

Reply:
I don’t know.